src/output/plugins/httpd/HttpdOutputPlugin.cxx

```diff
@@ -89,7 +89,7 @@ HttpdOutput::Unbind() noexcept
 inline void
 HttpdOutput::AddClient(UniqueSocketDescriptor fd) noexcept
 {
-	auto *client = new HttpdClient(*this, std::move(fd), GetEventLoop(),
+	auto client = std::make_unique<HttpdClient>(*this, std::move(fd), GetEventLoop(),
 				       !encoder->ImplementsTag());
 	clients.push_front(*client);
```

Comment:
This will crash. You create a new `HttpClient`, add it to the linked list, and upon leaving `AddClient()`, the `HttpdClient()` will be freed. What happens to the element still in the linked list?